src/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py

```diff
@@ -22,7 +22,7 @@
 from toil_scripts.lib.urls import download_url_job, s3am_upload
 from toil_scripts.tools.QC import run_fastqc
 from toil_scripts.tools.aligners import run_star
-from toil_scripts.tools.preprocessing import cutadapt
+from toil_scripts.tools.preprocessing import run_cutadapt
```

Comment:
Just one of those things I noticed while working on this PR, felt too unimportant to move to its own PR. Happy to remove it if you want me to.